worker.js

```diff
@@ -1 +0,0 @@
-require("./worker/main");
```

Comment:
worker files are now built into `dist/worker` and `npm serve` command points to `dist/worker/main.js`